spring-integration-scripting/src/test/java/org/springframework/integration/scripting/config/jsr223/Jsr223ServiceActivatorTests.java

```diff
@@ -123,6 +124,7 @@ public void withScriptVariableGenerator() throws Exception {
 	}
 
 	@Test
+	@EnabledOnJre(JRE.JAVA_8)
```

Comment:
What's wrong here, please ?

Reply:
```
java.lang.AssertionError
	at org.jruby.RubyModule.fetchConstant(RubyModule.java:4197)
	at org.jruby.RubyModule.getConstantAt(RubyModule.java:3681)
	at org.jruby.RubyModule.getConstantAt(RubyModule.java:3677)
	at org.jruby.javasupport.Java.getJavaPackageModule(Java.java:800)
	at org.jruby.javasupport.Java.getJavaPackageModule(Java.java:780)
	at org.jruby.javasupport.binding.ClassInitializer.initialize(ClassInitializer.java:47)
	at org.jruby.javasupport.binding.Initializer.setupProxyClass(Initializer.java:99)
```

Reply:
That's fine.
We may even remove all the Ruby tests.
I tried once to upgrade to the latest version and we have some conflicts with the Jython - they override some classes, but not fully :cry: 

I would say that JavaScript compatibility should be enough for us to trace: everything rest is just a target engine implementation and we are not responsible for their incompatibility with Java 10.

OK. Let's leave like you did. Maybe some comment on lines for the future consideration when Java 11 is going to be a base line ?